pkg/retrieval/retrieval_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package retrieval_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/p2p/mock"
+	"github.com/ethersphere/bee/pkg/p2p/protobuf"
+	"github.com/ethersphere/bee/pkg/retrieval"
+	storemock "github.com/ethersphere/bee/pkg/storage/mock"
+)
+
+type mockPeerSuggester struct {
+	returnValue string
+}
+
+func (v mockPeerSuggester) SuggestPeer(_ []byte) (string, error) {
+	return v.returnValue, nil
+}
+
+// TestDelivery tests that a naive request -> delivery flow works
```

Comment:
Use full sentences in comments (add dot at the end). https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences